sites/public/src/components/shared/MapControl.module.scss

```diff
@@ -0,0 +1,33 @@
+
+.map-control {
+  --control-border-color: var(--bloom-color-blue-900);
+  --control-border-radius: var(--doorway-border-100);
+
+  z-index: 1;
+  flex: 1;
+  flex-direction: column;
+  position: absolute;
+  top: var(--bloom-s8);
+  left: var(--bloom-s8);
+
+  .control-style {
+    background-color: var(--bloom-color-white);
+    border-color: var(--control-border-color);
+    padding: var(--bloom-s1);
+  }
+
+  .in-style {
+    border-width: var(--bloom-border-1) var(--bloom-border-1) calc(var(--bloom-border-1) / 2)
+      var(--bloom-border-1);
```

Comment:
Is it possible to make the border thicker? In the mocks the border is the same thickness as the plus and minus signs.

Reply:
I actually think the border should stay the same, but the svg on plus/minus need to change to smaller.  Look at my comments in Icons.tsx.

Reply:
I re-looked at this again and could be convinced as to what Ines is saying as well ... could make the borders thicker and it have relatively the same effect.